client/my-sites/checkout/checkout/index.jsx

```diff
@@ -453,12 +454,15 @@ export class Checkout extends React.Component {
 			return `/checkout/${ selectedSiteSlug }/add-support-session/${ receiptId }`;
 		}
 
+		if ( redirectTo ) {
+			return redirectTo;
```

Comment:
We need this after Gsuite purchase too. So, after signup, if user purchases a plan and a custom domain, then they're taken to the[ GSuite upsell](https://github.com/Automattic/wp-calypso/pull/29136/files?file-filters%5B%5D=.jsx#diff-199e3967ae413c2ff48db966abbf8386R430). 

Post GSuite purchase, the users are [redirected to Calypso preview](https://github.com/Automattic/wp-calypso/pull/29136/files?file-filters%5B%5D=.jsx#diff-199e3967ae413c2ff48db966abbf8386R395), but instead we need to redirect the users to the `redirectTo` value if it exists.

Reply:
It's also worth noting that the `Skip` button in both the GSuite upsell page and the Concierge upsell page need to respect the destination too.

Reply:
@niranjan-uma-shankar see https://github.com/Automattic/wp-calypso/pull/32410